draft-hsts-tracking-prevention.md

```diff
@@ -0,0 +1,256 @@
+---
+###
+# Internet-Draft Markdown Template
+#
+# Rename this file from draft-todo-yourname-protocol.md to get started.
+# Draft name format is "draft-<yourname>-<workgroup>-<name>.md".
```

Comment:
Can we follow this format? Something like `draft-sbingler-hstt-tracking-preventing.md`.

Reply:
sgtm

Reply:
sgtm